### PR TITLE
[misc] Add some links to README.md

### PR DESCRIPTION
**Pull request by archibate**
Related issue = #... (if any)

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,9 +44,19 @@ Please build from source for other configurations (e.g., your CPU is ARM).
...
+- [Taichi Conference](https://github.com/taichi-dev/taichicon): Taichi Online Virtual Conference available in both Chinese and English, approximately per-month.
+- [GAMES 201 Lectures](https://github.com/taichi-dev/games201): (Chinese) A hands-on tutorial on advanced physics engines based on Taichi, per-week.
+
+---
+
+- [Taichi Elements](https://github.com/taichi-dev/taichi_elements): A High-Performance Multi-Material Continuum Physics Engine based on Taichi (work in progress).
+- [LBM Taichi](https://github.com/hietwll/LBM_Taichi): Fluid solver based on Lattice Boltzmann method implemented by Taichi programming language, by [Zhuo Wang (hietwll)](https://github.com/hietwll).
+- [Shadertoy reproduced by Taichi](https://github.com/Phonicavi/Shadertoy-taichi): Some prevalent shadertoys implemented in Taichi, by [QIU Feng (Phonicavi)](https://github.com/Phonicavi).
+- [DiffTaichi](https://github.com/yuanming-hu/difftaichi): 10 differentiable physical simulators built with Taichi differentiable programming, by [Yuanming Hu (yuanming-hu)](https://github.com/yuanming-hu).
```
Comment (archibate) [inline README.md:59]: Upper 4 is platforms, Lower 4 is applications. `Taichi Elements` is by `Taichi community` right? Is `DiffTaichi` by @yuanming-hu or community? If community, please move it to taichi-dev too.

File: README.md [hunk 2]
```diff
@@ -44,9 +44,19 @@ Please build from source for other configurations (e.g., your CPU is ARM).
...
```
Comment (archibate) [inline README.md:60]:
```suggestion
- [Taichi GLSL](https://github.com/yuanming-hu/difftaichi): Manipulate Taichi with GLSL-alike functions / APIs.

```

File: README.md (hunk 1 above)
Comment (yuanming-hu) [reply README.md:59]: > Upper 4 is platforms, Lower 4 is applications. `Taichi Elements` is by `Taichi community` right? Is `DiffTaichi` by @yuanming-hu or community? If community, please move it to taichi-dev too.

Looks good! I'll try to move `difftaichi` to `taichi-dev` later this week.

File: README.md [hunk 3]
```diff
@@ -44,9 +44,20 @@ Please build from source for other configurations (e.g., your CPU is ARM).
...
+- [Taichi GLSL](https://github.com/yuanming-hu/difftaichi): Manipulate Taichi with GLSL-alike functions / APIs.
```
Comment (archibate) [inline README.md:60]:
```suggestion
- [Taichi GLSL](https://github.com/yuanming-hu/difftaichi): Manipulate Taichi with GLSL-alike helper functions.
```

**Pull request merged**